compiler-rt/lib/tsan/rtl/tsan_interceptors_posix.cpp

```diff
@@ -1088,7 +1088,18 @@ TSAN_INTERCEPTOR(int, pthread_join, void *th, void **ret) {
   return res;
 }
 
-DEFINE_REAL_PTHREAD_FUNCTIONS
+// DEFINE_REAL_PTHREAD_FUNCTIONS
+namespace __sanitizer {
+int real_pthread_create(void *th, void *attr, void *(*callback)(void *),
+                        void *param) {
+  ScopedIgnoreInterceptors ignore;
```

Comment:
Mmap interceptor clears shadow for the mmaped memory range. If we ignore mmaps coming from pthread_create (stacks), can't we get garbage shadow for these and report false positives later?

Reply:
I will rename real_pthread_create -> create_internal thread, or something in followup patch.

Reply:
Ah, I see.